.github/workflows/test_on_push.yaml

```diff
@@ -47,6 +53,10 @@ jobs:
           tarantool --version
           ./deps.sh
 
+      - name: Install metrics
+        if: matrix.metrics-version != ''
+        run: tarantoolctl rocks install metrics ${{ matrix.metrics-version }}
```

Comment:
We manage our dependencies in `deps.sh`, let's move metrics module to `deps.sh` too.

Reply:
Well, the integration is optional. It's not the same as luatest dependency. Moreover, I'm not sure we should build some wrapper to Github CI env vars to pass `matrix.metrics-version != ''` to `deps.sh script.